### ISSUE TITLE
Implement saccadic suppresion

### ISSUE THREAD

**Issue by ruipimentelfigueiredo** [vislab-tecnico-lisboa/move_robot#2]
Implement saccadic suppression here, since this module is the one responsible for commanding the head-eye movements. Several modules will benefit from this feature, including pedestrian detection and online stereo calibration. In the future the mechanism should be closer to the hardware

**Comment by ruipimentelfigueiredo** [comment]
@joao-avelino @plinioMoreno lets implement it here?  I would say in the Gaze.cpp and Gaze.h, since it should work both for simulation and for the real platform.

I think it's cleaner. For instance, Moutinho's online calibration also needs this. Basically,  suppression is always useful, no matter the application involving saccades. Visual-proprioceptive measurements acquired during saccades are of no use.

**Comment by plinioMoreno** [comment]
I think it's a good idea. Implementing at the hardware level creates more configuration issues.

**Comment by ruipimentelfigueiredo** [comment]
For now, I propose the following implementation:

Let the move_robot node subscribe the /robot/camera/image_raw topic and output the image in a different namespace, let's say, /robot/camera/suppression/image_raw. As the name suggests the output should be suppressed during head-eye movements. All the other  nodes (rectification, pedestrian detection, stereo calibration, etc) should subscribe from the suppression namespace.

So basically whenever a new fixation point goal arrives. We set a control flag that disables the  /robot/camera/suppression/image_raw publisher, while the suppression criteria is satisfied (for now, using velocities, as described in the BICA paper).

Do you agree?

**Comment by plinioMoreno** [comment]
Sounds good to me.
